modules/library/src/test/scala/zio/elasticsearch/QueryDSLSpec.scala

```diff
@@ -980,6 +1005,31 @@ object QueryDSLSpec extends ZIOSpecDefault {
 
           assert(query.toJson)(equalTo(expected.toJson))
         },
+        test("properly encode Nested Query with MatchAll Query and inner_hits") {
```

Comment:
We can use "inner hits".

Reply:
Should I replace all other instances of the snake case naming convention when referring to Elasticsearch parameters in this file as well?